### DISCUSSION TITLE
SH-ESP32 load problems

### DISCUSSION THREAD

**Discussion by wannathermal** [hatlabs/discussions#72] [Q&A]
I have tried to follow the instructions but see the following in the terminal. Any ideas?

ELF file SHA256: 0720504c742bc907

Rebooting...
ets Jul 29 2019 12:21:46

rst:0xc (SW_CPU_RESET),boot:0x17 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13232
load:0x40080400,len:3028
entry 0x400805e4
E (6949) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
E (6949) task_wdt:  - loopTask (CPU 1)
E (6949) task_wdt: Tasks currently running:
E (6949) task_wdt: CPU 0: IDLE
E (6949) task_wdt: CPU 1: loopTask
E (6949) task_wdt: Aborting.

abort() was called at PC 0x400e1acd on core 0

Backtrace: 0x4008388d:0x3ffbea6c |<-CORRUPTED

ELF file SHA256: 0720504c742bc907

Rebooting...
ets Jul 29 2019 12:21:46

rst:0xc (SW_CPU_RESET),boot:0x17 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13232
load:0x40080400,len:3028
entry 0x400805e4
E (6949) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
E (6949) task_wdt:  - loopTask (CPU 1)
E (6949) task_wdt: Tasks currently running:
E (6949) task_wdt: CPU 0: IDLE
E (6949) task_wdt: CPU 1: loopTask
E (6949) task_wdt: Aborting.

abort() was called at PC 0x400e1acd on core 0

Backtrace: 0x4008388d:0x3ffbea6c |<-CORRUPTED

**Comment by mairas** [comment]
Can you please describe what you are trying to do? What software you're trying to flash, what modifications have you done, what is your hardware setup, etc?

**Comment by wannathermal** [comment]
Matti, thanks for the quick response.

I have a Sailor Hat with the SH-ESP32 and downloaded the SH-ESP32-nmea2000-gateway-main package. No mods. Ran Platform IO seemingly without problems, upload came back as successful, The blinking blue light on the board is blinking slowly as per the instructions.

Bob

________________________________

**Reply by mairas** [reply]
What is your hardware setup like? Meaning, how is your NMEA 2000 network set up?

**Comment by wannathermal** [comment]
I don't have an nmea 2000 network setup here at home. I do have an N0183 I can set up. I was hoping to be able to do some learning before going to the boat (which is 5oonm away at this point.

Bob

________________________________

**Comment by wannathermal** [comment]
Ok, I have fixed the loading issue. Started over from scratch and loaded fine. SignalK server running on windows (waiting on hardware). Now I just have to figure out how to route my 0183 and seatalk connections through the Sailor hat!
